wscript

```diff
@@ -377,6 +377,12 @@ iconv support use --disable-iconv.",
         'deps': 'gpl',
         'func': check_true,
         'default': 'disable',
+    }, {
+        'name': '--dvbv5',
+        'desc': 'DVBv5 input module',
+        'deps': 'gpl',
```

Comment:
Is this correct? Arch has `v4-utils` (which contains libdvdv5) marked as LGPL

Reply:
[v4l-utils git](https://git.linuxtv.org/v4l-utils.git/tree/) and [Gentoo Packaging](https://packages.gentoo.org/packages/media-tv/v4l-utils) indicate the code base might contain some code parts with GPL and some with LGPL, but `COPYING.libdvbv5` indicates `LGPL-2.1+`. Checking some of the `libdvb` files, they also contain the LGPL header, but probably all should be checked to be sure.